1_Contenido/1_1__Requisitos.md

```diff
@@ -396,16 +396,165 @@ recursos disponibles.
 Los requerimientos no deben exceder el presupuesto, así que esta restricción
 determina qué requerimientos son incluidos en el producto.
 
-### Otros requerimientos del producto
+### Casos de uso del producto
 
-Utilizando el [template de requerimiento
-atómico](../3_Plantillas/3_1_Requerimiento_atomico.md), listar todos los
-estándares aplicables, requerimientos de plataformas o hardware, requerimientos
-de rendimiento y requerimientos ambientales.
+En esta subsección se definen los [casos de uso del
+producto](../4_Conceptos/4_Caso_de_uso_del_producto.md) y se detallan
+individualmente.
+
+#### Diagrama o tabla de casos de uso
+
+Mediante un diagrama de casos de uso o una tabla de casos de uso, definir los
+casos de uso del producto. La tabla de casos de uso suele ser más cómoda cuando
+la cantidad de casos de uso es mayor a 20.
+
+#### Detalles de los casos de uso del producto
+
+Definir los detalles de cada caso de uso del producto. Esto se puede realizar
+mediante un guión gráfico, un escenario descrito textualmente, un prototipo de
+baja o alta fidelidad, una especificación de casos de uso incluyendo excepciones
+y alternativas, o un diagrama de secuencia o actividades.
+
+### Requerimientos funcionales
+
+En esta subsección se detallan los requerimientos funcionales del producto,
+utilizando el [template de requerimiento
+atómico](../3_Plantillas/3_1_Requerimiento_atomico.md) para cada uno de ellos.
+
+Estos requerimientos emergen de los casos de uso del producto.
+
+### Requerimientos no funcionales
+
+En esta subsección se detallan los diferentes tipos de requerimientos no
+funcionales del producto, utilizando el [template de requerimiento
+atómico](../3_Plantillas/3_1_Requerimiento_atomico.md) en cada caso.
+
+#### Requerimientos de apariencia
+
+Detallar los requerimientos asociados a la apariencia del sistema, por ejemplo:
+guías de UX/UI a utilizar —quizá brindadas por el cliente—, marca corporativa de
+la organización cliente, qué colores utilizar, etcétera. Si el sistema no cuenta
+con ninguna interfaz gráfica de usuario, indicarlo.
+
+#### Requerimientos de usabilidad y humanidad
+
+Esta subsección trata con los requerimientos que hacen al producto usable y
+aceptable para los usuarios finales.
+
+##### Requerimientos de facilidad de uso
+
+Detallar las aspiraciones del cliente en cuanto a la facilidad de uso del

```

Comment:
```suggestion
Detallar los requerimientos del cliente en cuanto a la facilidad de uso del
```

Reply:
Porque aspiración puede quedar como "nice to have" y en ese caso no es estrictamente un requisito.